README.md

```diff
@@ -1,32 +1,41 @@
-# CrontabSublime
-version 0.1
+# Crontab for Sublime Text 3
 
-![Example](CrontabHighlightSample.png)
+version 1.0
 
-A package I wrote for ST3 to syntax highlight Crontabs since [the old package available](https://github.com/clarkewd/SublimeCrontab) has not been updated since 2015 and doesn't work. I have inherited a regex, but intend to remove it's use in a future release as it's quite complicate and doesn't allow TODO 1 easily.
+![Example screenshot][screenshot]
 
-Crontabs are arguably a dying thing with systemd, but crontabs are still a quick and dirty way to get things running without too much effort, so some use is nice.
+A package for ST3 to syntax highlight Crontabs since [the old package][clarkewd-cron] currently available has not been updated since 2015 and doesn't work.
+
+Crontabs are arguably a dying thing with systemd, but crontabs are still a quick and dirty way to get things running without too much effort, so some help is nice.
 
 ## Installation
 
-1. Run “Package Control: Install Package” command, find and install `Crontab` plugin.
-2. Close and reopen file
+1. Run “Package Control: Install Package” command, find and install the `Crontab` plugin.
+2. Close and reopen the file.
 
-## Notes
+## Features
 
-- Will highlight packages with `tab` and `crontab` suffices. For custom suffices/filenames, add to your personal sublime text preferences.
-- Uses bash highlighting in your commands.
-- Comment and uncommenting lines using `ctrl/cmd + /` works as expected
+- Syntax highlighting for `.tab` and `.crontab`, with embedded shell syntax for script contents.
```

Comment:
Split up the first line. Something like this? I think put the cron specific features first and drop the commenting (unless you think it's useful)

- Syntax highlighting for crontab files (default `*.tab`, `*.crontab` and `cron.d`)
- Color-coded underline for `cron` expressions.
- Completions for `cron` expressions and enums like month names.
- Hover on a `cron` expression for an explanation.
- Shell syntax highlighting for cron commands
- Comment and uncommenting lines using <kbd>Ctrl</kbd>+<kbd>/</kbd> or <kbd>Cmd</kbd>+<kbd>/</kbd>.